publish-extension.js

```diff
@@ -41,7 +41,9 @@ const { createVSIX } = require('vsce');
             await exec(`git checkout ${context.ref}`, { cwd: context.repo });
             if (extension.custom) {
                 try {
-                    await exec(extension.custom, { cwd: context.repo });
+                    for (const command of extension.custom) {
+                        await exec(command, { cwd: context.repo });
+                    }
```

Comment:
I'd recommend to allow both strings and arrays:
```suggestion
                    const commands = typeof extension.custom === "string" ? [extension.custom] : extension.custom;

                    for (const command of commands) {
                        await exec(command, { cwd: context.repo });
                    }
```